src/main/webapp/trips/tripScript.js

```diff
@@ -21,9 +21,86 @@
 $(document).ready(() => {
   // Set the content width of the site.
   setContentWidth('800px');
+
+  // Add the trip cards to the site.
+  getAndAddTripCards();
 });
 
 // Set the width of the content container.
 function setContentWidth(width) {
   document.getElementById('content').style.width = width;
 }
+
+// Fetch the trip details for the user from the database, add trip cards.
+function getAndAddTripCards() {
```

Comment:
I take my previous comment back I see that null is used here!

Reply:
Yes great, that's right! :)